### DISCUSSION TITLE
Minver should not use GIT tags to determine the version of a project

### DISCUSSION THREAD

**Discussion by Anthony-Breneliere** [adamralph/minver#497] [Ideas]
Using GIT tags makes the building process to depend on the GIT repository, so if I do not join the GIT repository within the build process, this makes the project unable to build with the correct version. We have an unallowed circular reference between the Git repository (directory .git) and the source repository, and only the Git repository is supposed to know about our source repository.

**Comment by adamralph** [comment]
The entire purpose of MinVer is to use Git tags to determine versions. If that doesn't suit your workflow, you should consider a different solution.

**Reply by Anthony-Breneliere** [reply]
Ok, thank you for advice
